### ISSUE TITLE
Firefox Speech Synthesis option should be enabled by default

### ISSUE THREAD

**Issue by Nyabinary** [NixOS/nixpkgs#259497]
Firefox's speech synthesis got added as support in #210453, but it's disabled by default. I'm suggesting here that since speech synthesis is an accessibility feature, it should be enabled by default and then opted out by the user if the user doesn't require or need speech synthesis.

**Comment by cafkafk** [comment]
https://github.com/NixOS/nixpkgs/pull/261058 might revert this, this is probably not solved before that one has been merged/rejected

**Comment by Nyabinary** [comment]
#261058 got closed.
